src/styles/cylc/_workflow.scss

```diff
@@ -4,10 +4,10 @@
 #workflow-panel {
   #main {
     display: flex;
-    min-height: calc(100vh - 82px - 56px);
```

Comment:
82px was the size of the footer.

56px seemed to be a bit wrong - not sure where that came from.

Reply:
>56px seemed to be a bit wrong - not sure where that came from.

:me hiding:

Reply:
hehe, extremely minor issue :+1: